### PR TITLE
1.21 release notes

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by tbrisker** [comment]
@mbacovsky - can you take a looks at the hammer commit please? i pulled in from redmine what was marked as fixed in 0.16.0 for both h-c and h-c-f but i did not verify it matches the actual commits in the projects.

File: _includes/manuals/1.21/1.2_release_notes.md [hunk 1]
```diff
@@ -4,9 +4,293 @@ This section will be updated prior to the next release.
...
+
+#### Hammer CLI
+* Update api doc used in foreman tests to match Foreman 1.20 ([#25376](https://projects.theforeman.org/issues/25376))
+* Report templates command ([#24489](https://projects.theforeman.org/issues/24489))
+* hammer template dump does not request password when output redirected to a file ([#14832](https://projects.theforeman.org/issues/14832))
+
+#### Hammer core
+* Add switch to hammer CLI to disable the defaults ([#25307](https://projects.theforeman.org/issues/25307))
+* Allow modification of output fields ([#21634](https://projects.theforeman.org/issues/21634))
+
```
Comment (mbacovsky) [inline _includes/manuals/1.21/1.2_release_notes.md:127]: Not sure what should be included in the release notes but this is the full list. I'll update the targets in redmine
```md
* Fixed instructions in i18n docs ([#25724](http://projects.theforeman.org/issues/25724))
* Option validators can be mixed with sources ([#22253](http://projects.theforeman.org/issues/22253))
* Validator uses options from option collector ([#19574](http://projects.theforeman.org/issues/19574))
* hammer -h does work with --output json, csv, yaml ([#25438](http://projects.theforeman.org/issues/25438))
* Update rpm names to use tfm ([PR #296](https://github.com/theforeman/hammer-cli/pull/296))
* Allows modification of output fields ([PR #275](https://github.com/theforeman/hammer-cli/pull/275)) ([#21634](http://projects.theforeman.org/issues/21634))
* Allow skipping message tests ([PR #292](https://github.com/theforeman/hammer-cli/pull/292)) ([#7451](http://projects.theforeman.org/issues/7451))
* Helper for editing text in editor ([PR #294](https://github.com/theforeman/hammer-cli/pull/294)) ([#24489](http://projects.theforeman.org/issues/24489))
* Add ability to disable defaults ([PR #293](https://github.com/theforeman/hammer-cli/pull/293)) ([#25307](http://projects.theforeman.org/issues/25307))
* Allow modification of additional help texts ([PR #277](https://github.com/theforeman/hammer-cli/pull/277)) ([#21633](http://projects.theforeman.org/issues/21633))
```

File: _includes/manuals/1.21/1.2_release_notes.md [hunk 2]
```diff
@@ -4,9 +4,293 @@ This section will be updated prior to the next release.
...
+#### Foreman modules
+* Avoid storing puppetserver version related parameters in the answer file ([#25839](https://projects.theforeman.org/issues/25839))
+* make taxonomy and login settings optional ([#25787](https://projects.theforeman.org/issues/25787))
+
+#### Hammer CLI
+* Update api doc used in foreman tests to match Foreman 1.20 ([#25376](https://projects.theforeman.org/issues/25376))
+* Report templates command ([#24489](https://projects.theforeman.org/issues/24489))
+* hammer template dump does not request password when output redirected to a file ([#14832](https://projects.theforeman.org/issues/14832))
+
```
Comment (mbacovsky) [inline _includes/manuals/1.21/1.2_release_notes.md:126]: The full change log for hammer-cli-foreman
```md
* Use make targets for translations from core ([#25724](http://projects.theforeman.org/issues/25724))
* Allow mixing option sources and validations ([PR #402](https://github.com/theforeman/hammer-cli-foreman/pull/402)) ([#22253](http://projects.theforeman.org/issues/22253))
* Include information about search fields ([#11431](http://projects.theforeman.org/issues/11431))
* IPv6 subnet option in hostgroup create/update ([#18752](http://projects.theforeman.org/issues/18752))
* tests for message formats ([#7451](http://projects.theforeman.org/issues/7451))
* Report templates - inputs, clone, interactive updates ([#24489](http://projects.theforeman.org/issues/24489))
* Update API doc in tests to match Foreman 1.20 ([#25376](http://projects.theforeman.org/issues/25376))
* add a command for report templates ([#24489](http://projects.theforeman.org/issues/24489))
* Re-add --thin into host list ([PR #395](https://github.com/theforeman/hammer-cli-foreman/pull/395)) ([#25349](http://projects.theforeman.org/issues/25349))
* Request password regardless of redirection ([#14832](http://projects.theforeman.org/issues/14832))
```

**Comment by tbrisker** [comment]
Thanks @mbacovsky, updated with the full hammer and hammer foreman notes.

**Review by ekohl — APPROVED** [review]
I did update some issues but I'm fine with regenerating later. We probably want to do so anyway when we're building RC1 packages to get the final list.

**Comment by tbrisker** [comment]
Thanks @ekohl, we'll definitely update this at least once before the actual release with further issues cherry-picked.

**Pull request merged**